README.md

```diff
@@ -1,76 +1,69 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Catalog of my things
 
-> One paragraph statement about the project.
+> This app helps users to keep a record of different types of things they own
 
-![screenshot](./app_screenshot.png)
+This is a simple console app that allows user to manage collections of books they own
 
-Additional description about the project and its features.
+This app can do the following:
 
-## Built With
+- Show a list of books
+- Show a list of labels
+- Store books
 
-- Major languages
-- Frameworks
-- Technologies used
+All the books and labels are stored in JSON files
 
-## Live Demo
+## Built With
 
-[Live Demo Link](https://livedemo.com)
+- Ruby
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- Ruby >= 3.0.1
+
 ### Setup
```

Comment:
[ OPTIONAL ] Please add the `bundle install` command in setup instructions so that non-technical users can install all the project dependencies.